### PR TITLE
S3 support FSBP

### PR DESCRIPTION
**Pull request by bbernays**
- [ ] Comments for complex S3 query

### CODE REVIEW

File: queries/s3/bucket_level_public_access_blocks.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+SELECT arn, region
+FROM AWS_S3_BUCKETS
+WHERE BLOCK_PUBLIC_ACLS IS NOT TRUE
```
Comment (yevgenypats) [inline queries/s3/bucket_level_public_access_blocks.sql:3]: Nit: table names and columns in lower case?

**Review by yevgenypats — APPROVED** [review]
LGTM. added nit comment.

File: queries/s3/bucket_level_public_access_blocks.sql (hunk 1 above)
Comment (bbernays) [reply queries/s3/bucket_level_public_access_blocks.sql:3]: Sure. This was me starting to play around with SQL linter/formatters

Comment (bbernays) [reply queries/s3/bucket_level_public_access_blocks.sql]: Done

**Pull request merged**